.github/workflows/developer-site-ci.yml

```diff
@@ -0,0 +1,45 @@
+name: Diem Developers Website CI
```

Comment:
this should likely live in ci-test.yml now, especially if the site not building on auto should prevent it landing on main.